### ISSUE TITLE
AppImage don't start

### ISSUE THREAD

**Issue by gelderode** [aardappel/treesheets#193]
## Configuration
```
OS: Fedora release 35 (Thirty Five) x86_64  
Kernel: 5.15.6-200.fc35.x86_64  
Shell: bash 5.1.8  
DE: Xfce 4.16 
WM: Xfwm4  
CPU: Intel i7-2635QM (8) @ 2.900GHz 
GPU: Intel 2nd Generation Core Processor Family 
GPU: AMD ATI Radeon HD 6400M/7400M Series 
Memory: 3674MiB / 7851MiB```
```
## AppImmage
`TreeSheets-5175451-x86_64.AppImage`
- Does nothing when clicked in de DesktopEnvironment
- When started in terminal following errors are given:
  - treesheets: /tmp/.mount_TreeShuCU9t6/usr/lib/libselinux.so.1: no version information available (required by /lib64/libgio-2.0.so.0)
  - treesheets: /tmp/.mount_TreeShuCU9t6/usr/lib/libselinux.so.1: no version information available (required by /lib64/libmount.so.1)
  - treesheets: symbol lookup error: /lib64/libgio-2.0.so.0: undefined symbol: g_module_open_full

## TARBALL
`treesheets_linux64.tar.gz`
- No problems, runs smoothly with all the possibilities

**Comment by aardappel** [comment]
I don't maintain the appimage and generally am not familiar with them, so not sure.

That said, it sounds like this is a common issue (searching for `undefined symbol: g_module_open_full`), e.g. https://bbs.archlinux.org/viewtopic.php?id=269838

**Comment by aardappel** [comment]
@probonopd

**Comment by Deevad** [comment]
Hey, same issue on Fedora 36 KDE:
```
 ~/sources »»» ./TreeSheets-5175451-x86_64.AppImage 
treesheets: /tmp/.mount_TreeShmpz1qM/usr/lib/libselinux.so.1: no version information available (required by /lib64/libgio-2.0.so.0)
treesheets: /tmp/.mount_TreeShmpz1qM/usr/lib/libselinux.so.1: no version information available (required by /lib64/libmount.so.1)
treesheets: symbol lookup error: /lib64/libgio-2.0.so.0: undefined symbol: g_module_open
```

(thanks OP for the Tar.Gz workaround :+1: )
